data/ui/preferences/collection.ui

```diff
@@ -62,5 +62,71 @@
         <property name="top_attach">3</property>
       </packing>
     </child>
+    <child>
+      <object class="GtkCheckButton" id="collection/write_rating_to_disk">
+        <property name="label" translatable="yes">Save ratings in files</property>
```

Comment:
Could we be a bit more clear with our wording here? If I'm not mistaken, ratings are already written to a file on disk in `music.db`. This feature would write ratings to the metadata of the audio files themselves, right? Maybe semantics like `collection/write_rating_to_audio_file` and `Save ratings in audio file metadata` would be more clear?